src/utilities.ts

```diff
@@ -395,6 +453,15 @@ export async function prepareDeliverOptionParams(
     version = DEFAULT_VERSION,
     wallet: ethers.Wallet | ethers.Signer
 ): Promise<DeliverOptionParams> {
+
+    // Ensure that the payPremium boolean is set for closing short position.
```

Comment:
I think you can copy this check and paste it on line 277 in `arrow-sdk` since they appear to be the same, but the one in `arrow-sdk` is a little old/wrong